src/main/resources/assets/techreborn/lang/en_us.json

```diff
@@ -1106,5 +1107,13 @@
   "gui.techreborn.block.output_blocked": "Output blocked",
   "gui.techreborn.block.progress.active": "Progress: (%s)",
   "gui.techreborn.block.progress.stopped": "Idle (Stopped)",
-  "gui.techreborn.block.progress.paused": "Idle (%s)"
+  "gui.techreborn.block.progress.paused": "Idle (%s)",
+
+  "_comment27": "Patchouli Book",
+  "patchouli.techreborn.guide.name": "Tech Reborn Manual",
+  "patchouli.techreborn.landing_text": "Welcome to the world of Tech Reborn.$(br2)Check out our online $(l:https://wiki.techreborn.ovh/doku.php)Wiki$()$(br2)You can also join our $(l:https://discord.gg/teamreborn)Discord!$()$(br2)If you have feedback, suggestions, issues or want to contribute visit our $(l:https://github.com/TechReborn/TechReborn/issues)GitHub$()$(br2)We hope you enjoy playing!",
+
+  "_comment28": "Guide Book Machines",
+  "patchouli.techreborn.machine.block_breaker": "The block breaker is a simple machine. It allows you to break a block from the location in front of it, to its inventory. This action costs energy",
+  "patchouli.techreborn.machine.block_placer": "The block placer is a simple machine. It allows you to place a block from its inventory, to the location in front of it. This action costs energy"
```

Comment:
Im unsure if this is the best way to document TR, the wiki is the go-to place for documentation, im unsure if we want to fragment it even more?

Reply:
This is totally wrong. Patchouli is worked on per translation. Ie rather than adding translation keys, you add a complete book in the correct language, which is explained on the patchouli book page as well.

Regarding the wiki, it is bad insofa that it is not being kept up to date. I want to do something with datagen to automatically keep the wiki and the patchouli up to date and synchronized

Reply:
Patchouli has two options for i18n

I opted for the optional Lang files support to allow the already working crowd source translation to work instead of having a new book for each language. (As suggested by the mod).

Reply:
Okay, I admit I didn't know that.
Nevertheless, this was makes maintaining the book much harder imo

Reply:
I agree, it is harder to create the book this way, but on the other hand it fragments the way translations are handled. I wanted to spark discussion, because I could not see anything else.